cypress/integration/e2e/end_to_end_spec.js

```diff
@@ -1,15 +1,135 @@
+        cy.get(this.element.appHeaderElements.newProjectButton)
+        .click();
+
+        cy.get(this.element.modalElements.modalProjectNameField)
+        .type(newProjectName);
+
+        cy.get(this.element.modalElements.modalSaveButton)
+        .click();
+
+        // Go to newly created team
+        cy.get(this.element.homePage.projectList)
+        .children()
+        .contains('.data-list-view__icon', 'CE')
+        .click();
+
+        cy.get(this.element.appHeaderElements.headerTitle)
+        .contains(newProjectName);
+
         // Add a team to the project
+        cy.get(this.element.appHeaderElements.projectNavigation)
+        .children()
+        .contains('span', 'TEAMS')
+        .click();
+
+        cy.get(this.element.teamPage.teamInputField)
+        .type(newTeamName)
+        .get(this.element.teamPage.addTeamButton)
+        .click();
+
         // Setup a sprint
+        cy.get(this.element.appHeaderElements.projectNavigation)
+        .children()
+        .contains('span', 'SPRINTS')
+        .click();
+
+        cy.get(this.element.sprintPage.sprintNumberField)
+        .type(1);
+
+        // cy.get(this.element.sprintPage.sprintStartDate)
+        // .type(startDate);
+
+        // cy.get(this.element.sprintPage.sprintEndDate)
+        // .type(endDate);
+
+        cy.get(this.element.homePage.projectListElement.buttons)
+        .click();
+        
         // Fill out health check
+        cy.get(this.element.appHeaderElements.projectNavigation)
+        .children()
+        .contains('span', 'TEAM HEALTH')
+        .click();
+
+        cy.get(this.element.healthPage.healthRadioButton)
+        .each(($el) => {
+            if(healthCount === 0) {
+                cy.wrap($el).click();
+                healthCount++;
+            }
+            else if(healthCount === 1) {
+                cy.wrap($el).click();
+                cy.wrap($el).click();
+                healthCount++;
+            }
+            else {
+                cy.wrap($el).click();
+                cy.wrap($el).click();
+                cy.wrap($el).click();
+                healthCount = 0;
+            }
+        });
+
+        // Save the changes
+        cy.get(this.element.healthPage.healthSaveButton)
+        .click();
+
+        //Reset the counter 
+        var healthColour = 0;
+
+        // Check each radio button is assigned the correct value
+        cy.get(this.element.healthPage.healthRadioButton)
+        .each(($el) => {
+            if(healthColour === 0) {
+                cy.wrap($el).should('have.css', 'background-color', 'rgb(255, 51, 0)');
+                healthColour++;
+            }
+            else if(healthColour === 1) {
+                cy.wrap($el).should('have.css', 'background-color', 'rgb(255, 204, 51)');
+                healthColour++;
+            }
+            else {
+                cy.wrap($el).should('have.css', 'background-color', 'rgb(0, 204, 102)');
+                healthColour = 0;
```

Comment:
Same with `healthColour` - what is it really? the name is a bit misleading when you're assigning integers to it